src/css/components/app-status.less

```diff
@@ -30,6 +30,12 @@
     color: @app-status-waiting-color;
   }
 
+  &.waiting-for-action {
```

Comment:
Precession nitpick.
How about:
`&.waiting-for-user-action`
?

Just an `action` could be everything.